examples/complete-mysql/main.tf

```diff
@@ -49,8 +49,10 @@ module "db" {
   backup_window                   = "03:00-06:00"
   enabled_cloudwatch_logs_exports = ["general"]
   create_cloudwatch_log_group     = true
+  blue_green_update_enabled       = true
 
-  backup_retention_period = 0
+  # Backups must be enabled for Blue Green Deployments
+  backup_retention_period = 1
```

Comment:
if this is required, then we should change this https://github.com/terraform-aws-modules/terraform-aws-rds/blob/2ef19638628266df0f6f7cc9a218c846e579023a/modules/db_instance/main.tf#LL84C3-L84C13 to 

```hcl
  backup_retention_period = length(var.blue_green_update) > 0 ? coalesce(var.backup_retention_period, 1) : var.backup_retention_period
```